app/comparator.py

```
@@ -0,0 +1,134 @@
+import subprocess
+import os
+import json
+from datetime import datetime
+from typing import Optional, Tuple
```

Ruff (0.8.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)